components/Layout.tsx

```diff
@@ -0,0 +1,17 @@
+import Header from "./Header";
+import Navigation from "./Navigation";
+import { ReactNode } from "react";
+
+interface AuthProps {
+  children: ReactNode;
+}
```

Comment:
As with most TypeScript solutions: When I see it, it makes total sense, but I wouldn't have thought that this is the solution. 😄